torchvision/io/image.py

```diff
@@ -186,21 +186,50 @@ def encode_jpeg(input: torch.Tensor, quality: int = 75) -> torch.Tensor:
     of its corresponding JPEG file.
 
     Args:
-        input (Tensor[channels, image_height, image_width])): int8 image tensor of
-            ``c`` channels, where ``c`` must be 1 or 3.
+        input (Tensor[channels, image_height, image_width]): uint8 image tensor of ``c`` channels, where ``c`` must be 1 or 3
         quality (int): Quality of the resulting JPEG file, it must be a number between
             1 and 100. Default: 75
 
     Returns:
-        output (Tensor[1]): A one dimensional int8 tensor that contains the raw bytes of the
-            JPEG file.
+        output (Tensor[1]): A one dimensional uint8 tensor that contains the raw bytes of the JPEG file.
     """
     if not torch.jit.is_scripting() and not torch.jit.is_tracing():
         _log_api_usage_once(encode_jpeg)
     if quality < 1 or quality > 100:
         raise ValueError("Image quality should be a positive number between 1 and 100")
+    if input.device.type == "cuda":
+        output: torch.Tensor = torch.ops.image.encode_jpeg_cuda([input], quality)[0]
+    else:
+        output: torch.Tensor = torch.ops.image.encode_jpeg(input, quality)
+
+    return output
+
+
+def encode_jpegs(inputs: list[torch.Tensor], quality: int = 75) -> list[torch.Tensor]:
```

Comment:
Hey Nicolas, thanks for taking a look!
I used a devGPU and didn't use buck or any internal tools. The PR builds and tests fine locally without any errors. I'm guessing this is because I'm using Python 3.11, but this test uses 3.8 (subscript for standard containers was introducted in PEP 585 which was released with python 3.9). Should be an easy fix.